### DISCUSSION TITLE
Playlists location

### DISCUSSION THREAD

**Discussion by RafaelEstevamReis** [rern/rAudio#272] [Testing]
Why does:
 rAudio stores playlists in `/srv/http/data/playlists/` 
And MPD is configured to `/var/lib/mpd/playlists"` ?

Can I safely change MPD configuration to use the same folder as rAudio ?
There is a reason to keep two folders ?

PS1: Why am I asking?: I use MPD tpc commands to automate other song equipment and synchronize different players, and now i'm migrating to rAudio
PS2: About the Category choice: I'm only asking for better rAudio understanding, so I used "testing"
PS3: I'm talking about the MPD configuration at `/etc/mpd.conf`

**Comment by rern** [comment]
`/var/lib/mpd/playlists` is for those who requested to use MPD clients other than rAudio browser based.

Anybody who knows why they're preferred please share.

**Reply by lochbika** [reply]
I prefer the mpd playlist location. Because the playlists are

1.  recognized by non Raudio mpd clients
2. mpd playlists support all features that are required by rune audio

https://www.musicpd.org/doc/html/plugins.html#playlist-plugins

In my opinion, there is no reason for a separate playlist format.

**Comment by RafaelEstevamReis** [comment]
> Can I safely change MPD configuration to use the same folder as rAudio ?

No, I can't

rAudio playlist is a different file format. It is not a m3u file nor anything, it is a json with some music info
